app/policies/impersonation_policy.rb

```diff
@@ -1,9 +1,13 @@
 class ImpersonationPolicy < ApplicationPolicy
   def create?
-    user.third_line_role?
+    return false if user.is_computacenter?
```

Comment:
Is there a way that third_line roles with computacenter access could still be permitted to do this?
I know there are a few of us...

Reply:
There's only a handful of users that fit your criteria. I don't recommended enabling the `is_computacenter` flag personally as it hides items from support views when you probably want to be able to see them. Personally I only toggle this flag if and when i need it